src/utils/send-request.js

```diff
@@ -106,7 +106,7 @@ function requestAPI (config, options, callback) {
   delete options.qs.followSymlinks
 
   const method = 'POST'
-  const headers = {}
+  const headers = config.headers || {}
```

Comment:
We need an `Object.assign({}, config.headers)` to avoid mutating the headers object passed as config, otherwise subsequent requests will receive the same headers as previous.